test/integration/examples/apiserver_test.go

```diff
@@ -90,9 +91,15 @@ func TestAggregatedAPIServer(t *testing.T) {
 			"--kubeconfig", wardleToKASKubeConfigFile,
 		})
 		if err := wardleCmd.Execute(); err != nil {
-			t.Fatal(err)
+			errs <- err
```

Comment:
I am slightly not sure if this is a better approach in this case, I suspect you're more likely to end up sending to a closed channel than you are receiving a useful error.  Do any other reviewers have an opinion?

Reply:
Right, i'll instead store the err directly without chececking if its nil, ill do that after i take it from channel